draft-packages/form/KaizenDraft/Form/Primitives/Checkbox/styles.scss

```diff
@@ -40,38 +41,38 @@ $focus-ring-offset: 2px;
   width: $checkbox-size;
   top: 0;
   border: $kz-border-solid-border-width $kz-border-solid-border-style
-    $kz-color-wisteria-500;
+    $dt-color-form-border-color;
   border-radius: $kz-border-solid-border-radius;
   box-sizing: border-box;
-
-  .checkbox:not([checked]):focus + & {
-    border-color: $kz-color-wisteria-700;
-    background-color: $kz-color-wisteria-100;
-  }
-
-  @media (hover: hover) and (pointer: fine) {
-    .checkbox:not([checked]):hover + & {
-      border-color: $kz-color-wisteria-700;
-      background-color: $kz-color-wisteria-100;
-    }
-  }
+  background-color: white;
+  border-color: $dt-color-form-border-color;
 
   .checkbox:checked + &,
   .checkbox:indeterminate + & {
-    background: $kz-color-wisteria-500;
+    background: $dt-color-form-border-color;
+    background-color: $dt-color-form-border-color;
```

Comment:
I think this is a typo and should be `border-color` ?

Reply:
ack, needed a new token. Fixed now.